Jenkinsfile

```diff
@@ -204,6 +204,7 @@ pipeline {
 
                     rm -fr analyzer/out/results
                     /opt/ort/bin/ort $LOG_LEVEL analyze $ALLOW_DYNAMIC_VERSIONS_PARAM $USE_CLEARLY_DEFINED_CURATIONS_PARAM -f JSON,YAML -i $PROJECT_DIR/source -o analyzer/out/results
+                    ln -fs analyzer/out/results/analyzer-result.yml $PROJECT_DIR/current-result.yml
```

Comment:
Commit message nit: "...point**s** to..."

Reply:
Done.